### ISSUE TITLE
[Build] build fail in v3.4.2

### ISSUE THREAD

**Issue by jackyzy823** [standardnotes/desktop#598]
I'm trying to build v3.4.2, but encounter some issues.

The dependency `"standard-notes-web`  in `app/package.json` may not work.

For example, even i try to 
```
npm install github:standardnotes/web#8a0b0c972f41e131f542ad18cc7bd547cb1aa4be
```

It results
```
npm ERR! code 128
npm ERR! Command failed: git checkout 8a0b0c972f41e131f542ad18cc7bd547cb1aa4be
npm ERR! fatal: reference is not a tree: 8a0b0c972f41e131f542ad18cc7bd547cb1aa4be
npm ERR!
```

This commit `8a0b0c972f41e131f542ad18cc7bd547cb1aa4be` can be found in  https://github.com/standardnotes/web/commit/8a0b0c972f41e131f542ad18cc7bd547cb1aa4be . However it is marked as `This commit does not belong to any branch on this repository.`  and `git checkout 8a0b0c972f41e131f542ad18cc7bd547cb1aa4be` doesn't work too.

It would be appreciated if you could update this dependency to a right commit. Thanks.

**Comment by jackyzy823** [comment]
Another issue in build in `develop` branch.

when running `npm run build`
it results
```
TypeError: merge is not a function
    at /desktop/webpack.prod.js:5:3
    at Array.map (<anonymous>)
    at Object.<anonymous> (/desktop/webpack.prod.js:4:27)
    at Module._compile (/desktop/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:194:30)
```

so line in https://github.com/standardnotes/desktop/blob/7dd015afeeabb9f6556e024289347c4bf5c424a8/webpack.prod.js#L1

should be

```
 const { merge } = require('webpack-merge'); 
```


Reference:https://stackoverflow.com/questions/62851934/webpack-merge-merge-is-not-a-function

**Comment by JaspalSuri** [comment]
Hi @jackyzy823, I've assigned our Desktop and Web app dev to look into this and they will get back to you.

**Comment by baptiste-grob** [comment]
Hey there, you've caught us right at the time when we're finishing up some infrastructure changes on `develop`. You can expect a commit that fixes this today or tomorrow at the latest.

**Comment by danielhass** [comment]
@baptiste-grob , running into the same issue with users of my AUR package:
https://aur.archlinux.org/packages/standardnotes-desktop
Any news on this one?

**Comment by baptiste-grob** [comment]
We're planning to have a v3.4.3 out sometime this week. I'll make sure it builds from a fresh install and let you know when it's ready.

**Comment by baptiste-grob** [comment]
You should be able to build from `master` now 👍

**Comment by danielhass** [comment]
@baptiste-grob - I can confirm that from my side. build + packing with removed patches via PKGBUILD works on my side.
Looking forward to the new release tag. Afterwards I will upgrade the AUR package accordingly.

**Comment by baptiste-grob** [comment]
Thanks for waiting on the tag :)
